docs/10-MultiPageDashApp-MetaTags.md

```diff
@@ -0,0 +1,147 @@
+
+> ## Status: Multi-Page Dash App Plugin
+> #### Under active development:  A plugin to simplify creating multi-page Dash apps. This is a preview of functionality that will be added to Dash 2.x.
+> **[See the community announcement for details and discussion](https://community.plotly.com/t/introducing-dash-pages-dash-2-1-feature-preview/57775)**
+
+
+# Multi-Page Dash App Examples
+
+**Please see Chapter 08-MultiPageDashApp for an introduction to the Multi-Page Dash App Plugin**
+
+
+### Example: Social Media Meta Tags
+  
+One of the nice features of this API is that it automatically creates the meta 
+tags used by social media sites like Facebook and Twitter.  These sites use the app title, description and image to create
+the card that is displayed when you share a link to your site. They are also used in search engine results.
+
+Find more information on social media meta tags [here.](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML)
```

Comment:
avoid links directly to "here" 🙂  https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/